<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duff - Duplicate File Finder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelSelectedDirectory">
      <property name="text">
       <string>Selected Directory:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditSelectedDirectory"/>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTreeView" name="treeViewResults"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonDeleteSelected">
      <property name="text">
       <string>Delete Selected</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>593</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButtonFindDuplicates">
      <property name="text">
       <string>Find Duplicates</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBarMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicenses"/>
   </widget>
   <widget class="QMenu" name="menuAlgorithm">
    <property name="title">
     <string>Algorhitm</string>
    </property>
    <addaction name="actionMD5"/>
    <addaction name="actionSHA_1"/>
    <addaction name="actionSHA_256"/>
    <addaction name="actionSHA_512"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAlgorithm"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionLicenses">
   <property name="text">
    <string>Licenses</string>
   </property>
  </action>
  <action name="actionMD5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MD5</string>
   </property>
  </action>
  <action name="actionSHA_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SHA-1</string>
   </property>
  </action>
  <action name="actionSHA_256">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SHA-256</string>
   </property>
  </action>
  <action name="actionSHA_512">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SHA-512</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
